src/golang/cmd/server/handler/connect_integration.go

```diff
@@ -202,58 +207,241 @@ func (h *ConnectIntegrationHandler) Perform(ctx context.Context, interfaceArgs i
 		// This integration should be used as the new storage layer.
 		// In order to do so, we need to migrate all content from the old store
 		// to the new store. This requires pausing the server and then restarting it.
-		// All of this logic is performed asynchronously so that the user knows that
-		// the connect integration request has succeeded and that the migration is now
-		// under way.
+
```

Comment:
Should we put these to a designated `lib` package like `storage_migration`, and call stuffs like `storage_migration.Prepare()`, `storage_migration.Migrate()`?

Reply:
I refactored the migration code into it's own file. Lemme know if that works.